src/GitHub.App/ViewModels/GitHubPane/PullRequestListItemViewModel.cs

```diff
@@ -31,6 +32,9 @@ public PullRequestListItemViewModel(PullRequestListItemModel model)
         /// <inheritdoc/>
         public IActorViewModel Author { get; protected set; }
 
+        /// <inheritdoc/>
```

Comment:
Actually, I'm not sure why I made these properties `protected set` - I think that was a mistake... I think I initially wrote this as a base class for both issues and PRs and then decided not to do that.